cernopendata/modules/fixtures/data/docs/delphi-guide-eventserver/delphi-guide-eventserver.md

```diff
@@ -0,0 +1,36 @@
+# Raw data access: the DELPHI event server
```

Comment:
The first title heading  is not necessary, since the page title will be automatically inserted as the first title heading based on the title found in `delphi-guide-eventserver.json`.